src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationResultEnrichmentService.kt

```diff
@@ -169,6 +174,15 @@ class CalculationResultEnrichmentService(
     emptyList()
   }
 
+  private fun hdc365Hints(hdcRules: Set<CalculationRule>, hdcedDate: LocalDate, hints: MutableList<ReleaseDateHint>) {
```

Comment:
The content of these hints is yet to be checked and confirmed by a Content Designer, they will be changed in a future ticket